File: Java/ProjectEuler/src/PE/Problem1.java
package PE;

/**
 * Created by Daniel on 2015-08-23.
 */
public class Problem1 {
    public static void main(String[] Args){
        int sum= 0 ;
        for(int i = 1;i<1000;i++) {
                if(i%5==0 || i%3==0){
                    sum+=i;
                }
            }
        System.out.println(sum);
        }

    }

File: CSharp/ProjectEuler/ProjectEuler/Problem1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProjectEuler
{
    class Problem1
    {
        public void run()
        {
            Console.WriteLine("The sum equals : " + Enumerable.Range(1, 999).Sum(x => (x % 3 == 0) ? x : (x % 5 == 0) ? x : 0));

           /* Old code, working like that one above  
            int sum = 0;
            int[] array = Enumerable.Range(1, 999).ToArray();
            foreach (int iterator in array)
            {
                if (iterator % 3 == 0 || iterator % 5 == 0)
                {
                    sum += iterator;
                }
            }
            Console.WriteLine("Done and the sum equals : "+sum); */
        }
    }
}
